.github/workflows/release.yml

```diff
@@ -52,22 +84,39 @@ jobs:
           pip install yq
 
           cd ${GITHUB_WORKSPACE}
-      - name: Build image, generate deployments and push git tag
+
+      - name: Release
+        env:
+          DWO_REPO: git@github.com:sleshchenko/devworkspace-operator
```

Comment:
Looks like you still have your repo and images setup, but other then that it looks good to me

Reply:
Good catch!